apps/app/pages/events.js

```diff
@@ -11,11 +11,13 @@ const { Title } = Typography;
 function Events() {
   const { data: events, isLoading } = useEvents();
 
+  let reversedEvents = events ? events.slice().reverse() : [];
```

Comment:
```suggestion
  let reversedEvents = events ? events.slice().reverse() : [];
```
Why do we need the `slice` method here?